packages/database/src/migrations/20210216073210-AddFetpDashboardGradByDistrict-modifies-data.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+import { arrayToDbString, insertObject } from '../utilities/migration';
+
+var dbm;
+var type;
+var seed;
+
+const report = {
+  id: 'FETP_PG_graduate_by_district',
```

Comment:
Super nit picky, but don't we normally do dashboard ids in all caps? Or have the times changed?

Reply:
Not sure when that was a convention - but the overwhelming majority of ids seem to be mixed case. Tried to follw convention of project country code prefix.